pom.xml

```diff
@@ -79,13 +79,13 @@
 		<!-- dependency versions -->
 		<junit.version>4.13.2</junit.version>
 		<storm-client.version>2.6.0</storm-client.version>
-		<jackson.version>2.15.2</jackson.version>
```

Comment:
Given that Storm is on 2.15.2 - probably best to stay on that as well

Reply:
2.16.x should be compatible.

Reply:
I have had too many issues with incompatibilities in Jackson over the years e.g class not found etc...
Will just use whatever Storm uses